packages/gasket-plugin-nextjs/lib/create.js

```diff
@@ -0,0 +1,145 @@
+ * createTestFiles
+ * @property {Files} files - The Gasket Files API.
+ * @property {generatorDir} - The directory of the generator.
+ * @property {testPlugin} - Selected test plugin from prompt
+ */
+function createTestFiles({ files, generatorDir, testPlugin }) {
+  const frameworks = ['jest', 'mocha', 'cypress'];
+  frameworks.forEach((tester) => {
+    const regex = new RegExp(`${tester}`);
+    if (regex.test(testPlugin)) {
+      files.add(`${generatorDir}/${tester}/*`, `${generatorDir}/${tester}/**/*`);
+    }
+  });
+}
+
+
+/**
+ * createNextFiles - Add next.config.js & server.mjs to files
+ * @property {Files} files - The Gasket Files API.
+ * @property {generatorDir} - The directory of the generator.
+ */
+function createNextFiles({ files, generatorDir }) {
+  files.add(`${generatorDir}/next/*`);
+}
+
+/**
+ * configureSitemap
+ * @property {Files} files - The Gasket Files API.
+ * @property {PackageJsonBuilder} pkg - The Gasket PackageJson API.
+ * @property {generatorDir} - The directory of the generator.
+ */
+function configureSitemap({ files, pkg, generatorDir }) {
+  pkg.add('dependencies', {
+    'next-sitemap': '^3.1.29'
+  });
+
+  files.add(`${generatorDir}/sitemap/*`);
+  pkg.add('scripts', {
+    sitemap: 'next-sitemap'
+  });
+}
+
+
+/**
+ * addDependencies
+ * @property {PackageJsonBuilder} pkg - The Gasket PackageJson API.
+ */
+function addDependencies({ pkg }) {
+  pkg.add('dependencies', {
+    '@gasket/assets': devDependencies['@gasket/assets'],
+    '@gasket/nextjs': devDependencies['@gasket/nextjs'],
+    'next': devDependencies.next,
+    'prop-types': devDependencies['prop-types'],
+    'react': devDependencies.react,
+    'react-dom': devDependencies['react-dom']
+  });
+
+  pkg.add('devDependencies', {
+    nodemon: devDependencies.nodemon
+  });
+}
+
+/**
+ * addRedux
+ * @property {Files} files - The Gasket Files API.
+ * @property {PackageJsonBuilder} pkg - The Gasket PackageJson API.
+ * @property {generatorDir} - The directory of the generator.
+ */
+function addRedux({ files, pkg, generatorDir }) {
+  pkg.add('dependencies', {
+    'next-redux-wrapper': devDependencies['next-redux-wrapper'],
+    'lodash.merge': devDependencies['lodash.merge']
+  });
+
+  files.add(`${generatorDir}/redux/*`, `${generatorDir}/redux/**/*`);
+}
+
+/**
+ * addNpmScripts
+ * @property {PackageJsonBuilder} pkg - The Gasket PackageJson API.
+ * @property {nextServerType} - Selected server type from prompt
+ */
+function addNpmScripts({ pkg, nextServerType }) {
+  const scripts = {
+    'next-cli': {
+      'build': 'next build',
+      'start': 'next start',
+      'start:local': 'next start & GASKET_ENV=local node server.mjs',
+      'local': 'next dev & nodemon server.mjs'
+    },
+    'next-custom': {
+      'build': 'next build',
+      'start': 'next build && node server.mjs',
+      'start:local': 'next build && GASKET_ENV=local node server.mjs',
+      'local': 'GASKET_DEV=1 GASKET_ENV=local nodemon server.mjs'
+    }
```

Comment:
With `createContext.nextServerType`, make your prompt values and keys here clearer

```suggestion
    defaultServer: {
      'build': 'next build',
      'start': 'next start',
      'start:local': 'next start & GASKET_ENV=local node server.mjs',
      'local': 'next dev & nodemon server.mjs'
    },
    customServer: {
      'build': 'next build',
      'start': 'next build && node server.mjs',
      'start:local': 'next build && GASKET_ENV=local node server.mjs',
      'local': 'GASKET_DEV=1 GASKET_ENV=local nodemon server.mjs'
    }
```